k8s/backend-deployment.tf

```diff
@@ -34,7 +34,7 @@ resource "kubernetes_deployment" "backend" {
                     }
                     
                     image_pull_policy 	= "Always"
-                    command             = ["/usr/bin/supervisord"]
+                    command             = ["/bin/bash", "docker/prod/start_app.sh"]
```

Comment:
I am trying to locate the path of file ```start_app.sh``` to see it's content. Please how do I locate it?

Reply:
It's on the backend repo